main.cpp

```diff
@@ -96,40 +98,50 @@ int main()
     bool closeWindow(House::Kitchen myKitchen);
 };
 
+House::House()
+{
+    std::cout << "House Constructed" << std::endl;
+    std::cout << std::endl;
+}
+
+House::Kitchen::Kitchen()
+{
+    numKnives = 10;
+    std::cout << "Kitchen Constructed" << std::endl;
+    std::cout << std::endl;
+}
+
 void House::Kitchen::microwaveChicken( float microwaveTime)
 {
-    float instructions = 45.f;
-    microwaveTime = instructions;    
-    
     if( microwaveTime >= 45.f )
     {
-        std::cout << "Ready!" << std::endl;
+        std::cout << "Cooked for " << microwaveTime << " seconds:" << " Chicken cooked!" << std::endl;
     }
     else
     {
-        std::cout << "Not Ready!" << std::endl;
+        std::cout << "Cooked for " << microwaveTime << " seconds:" << " Chicken not finished cooking!" << std::endl;
     }
 }
 
-bool House::Kitchen::madeSalad( std::string lettuceBrand, bool includeTomatoes, std::string dressing)
+bool House::Kitchen::madeSalad(std::string lettuceBrand, bool includeTomatoes, std::string dressing)
 {
-    lettuceBrand = "Good Lettuce";
-
-    if( includeTomatoes == true && dressing == "Italian" )
+    if( includeTomatoes == true && dressing == "Italian" && lettuceBrand == "Lettuce!" )
     {
+        std::cout << "Made salad!" << std::endl;
         return true;
     }
     else
     {
+        std::cout << "Didn't make salad" << std::endl;
         return false;
     }
 }
 
 float House::Kitchen::preheatOven(float ovenTemperature)
 {
-    ovenTemperature = 350.f;
     if (ovenTemperature > 0.f)
```

Comment:
You've got the same `return` issue as explained above in this function.
- [ ] revise